tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -817,6 +836,32 @@ def _test_cpu_cpuid_restore(context):
     microvm_factory = context.custom["microvm_factory"]
     cpu_template = context.custom["cpu_template"]
 
+    cpu_template_dir = cpu_template if cpu_template else "none"
```

Comment:
This piece looks very similar to the previous one. Would it make sense to extract it into a function?

Reply:
there was 1 other place where we just had  (cpu_template if cpu_template else "none"), so I created get_cpu_template_dir() to replace them all

Reply:
yeah, I meant also the comment with the explanation. If one updates that, they need to be aware of the second instance too.

Reply:
done, moved the common code to skip_test_based_on_artifacts()